openassessment/templates/openassessmentblock/student_training/student_training.html

```diff
@@ -30,7 +30,8 @@ <h2 class="step__title">
           <span class="step__status__label">{% trans "This step's status" %}:</span>
           <span class="step__status__value">
               <span class="copy">{% trans "In Progress" %}
-              (<span class="step__status__value--completed">{{ training_num_completed }}</span> of
+              (<span class="step__status__value--completed">{{ training_num_completed }}</span>
+                {% trans "of" %}
```

Comment:
Actually it is possible to do so, but I created this PR initially to address the issue temporarily [in my repo](https://github.com/Edraak/edx-platform/pull/202). 

I'm planning to fix this in the next upcoming days.

However, I'm facing a problem to get the ORA2 devstack running. I will post the details in the edxops-group.